core/src/main/java/org/apache/iceberg/ManifestEntriesTable.java

```diff
@@ -126,10 +126,10 @@ public long targetSplitSize() {
     @Override
     public CloseableIterable<StructLike> rows() {
       if (manifest.content() == ManifestContent.DATA) {
-        return CloseableIterable.transform(ManifestFiles.read(manifest, io).project(fileSchema).entries(),
+        return CloseableIterable.transform(ManifestFiles.read(manifest, io, null, null).project(fileSchema).entries(),
```

Comment:
Again, keep the original signatures as well

Reply:
Done